### PR TITLE
[zh] Translate Ambient in-Pod blog into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Translate Ambient in-Pod blog into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+以及提供内部 CNI 实现的服务，例如 [OpenShift](https://github.com/istio/istio/issues/42341)
+和 [Amazon EKS](https://github.com/istio/istio/issues/42340) 中使用 Ambient 模式。
+在任意 Kubernetes 中提供广泛支持已经成为 Ambient 网格转向 Beta
+版的首要要求 - 人们开始期望 Istio 能够在任意 Kubernetes 平台和任何 CNI 实现中工作。
+毕竟，如果 Ambient 不能随处可用，那么 Ambient 就不能被称为 Ambient！
+
+在 Solo 公司，我们一直在将 Ambient 模式集成到 Gloo Mesh 产品中，
+并针对这个问题提出了创新的解决方案。我们决定在
+2023 年末将我们的修改发布到[上游](https://github.com/istio/istio/issues/48212)，
+以帮助 Ambient 更快地达到 Beta 版，以便更多用户可以在 Istio 1.21 或更高版本中操作 Ambient，
```
Comment (windsonsea) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:30]:
```suggestion
以帮助 Ambient 更快进阶至 Beta，让更多用户可以在 Istio 1.21 或更高版本中用上 Ambient，
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+并在其平台中享受 Ambient 无 Sidecar 网格的优势，无论其现有或首选的 CNI 是如何实现的。
```
Comment (windsonsea) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:31]:
```suggestion
并在他们各自的平台中体验 Ambient 这种无 Sidecar 网格的优势，不用再操心现有的 CNI 或首选的 CNI 是什么。
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+
+## 我们是如何到达这里的？ {#how-did-we-get-here}
```
Comment (windsonsea) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:33]:
```suggestion
## 回首来时路，披荆斩棘 {#how-did-we-get-here}
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+
+### 服务网格和 CNI：非常复杂 {#service-meshes-and-cnis-its-complicated}
```
Comment (windsonsea) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:35]:
```suggestion
### 服务网格和 CNI：错综复杂 {#service-meshes-and-cnis-its-complicated}
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+
+Istio 是一种服务网格，而所有服务网格严格意义上来说都不是
+**CNI 实现** - 服务网格依赖[符合规范的主要 CNI 实现](https://www.cni.dev/docs/spec/#overview-1)存在于每个
```
Comment (windsonsea) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:38]:
```suggestion
**CNI 实现** ，服务网格想要在所有 Kubernetes 集群运行，
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+Kubernetes 集群中，并以此为基础。
```
Comment (windsonsea) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:39]:
```suggestion
底层需要一个[合规的主要 CNI 实现](https://www.cni.dev/docs/spec/#overview-1)。
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 7]
```diff
@@ -0,0 +1,264 @@
...
+
+此主要 CNI 实现可能由您的云供应商（AKS、GKE 和 EKS 都自带）提供，
```
Comment (windsonsea) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:41]:
```suggestion
这个主要 CNI 实现可能由您的云供应商（AKS、GKE 和 EKS 自己）提供，
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 8]
```diff
@@ -0,0 +1,264 @@
...
+也可能由 Calico 和 Cilium 等第三方 CNI 实现提供。
+一些服务网格还可能与它们自己的主要 CNI 实现捆绑在一起，它们明确要求这些实现才能够正常运行。
```
Comment (windsonsea) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:43]:
```suggestion
某些服务网格还可能捆绑了他们自己的主要 CNI 实现，他们明确要求只有这些实现才能让网格正常运行。
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 9]
```diff
@@ -0,0 +1,264 @@
...
+
+{{< image width="100%"
+    link="./traffic-flows-between-pods-in-ambient.svg"
+    alt="HBONE 流量在 Ambient 网格中 Pod 之间的流程"
+    >}}
+
+而且，和以前一样，对于有必要的用例，仍然可以处理来自网格外部的未加密的明文流量并强制执行策略：
+
+{{< image width="100%"
+    link="./traffic-flows-plaintext.svg"
```
Comment (windsonsea) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:233]: need a plan to localize these images

**Review by windsonsea — COMMENTED** [review]
great job for such a long blog. Need much more time to review.

**Comment by windsonsea** [comment]
See preview: https://deploy-preview-14564--preliminary-istio.netlify.app/latest/zh/blog/2024/inpod-traffic-redirection-ambient/

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md (hunk 9 above)
Comment (wilsonwu) [reply content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:233]: Yes, plan to do it a little later.

Comment (windsonsea) [reply content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:233]: Great, that can be included in a separate PR.

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 10]
```diff
@@ -0,0 +1,264 @@
...
+然而，当时的条件十分有限。Ambient 模式依赖于在工作负载 Pod 和
+[ztunnel](/zh/blog/2023/rust-based-ztunnel/) 之间以透明方式重定向流量，
+然而，最初为此使用的机制与多种第三方容器网络接口（CNI）实现相冲突。
+我们在 GitHub Issue 和 Slack 上反复讨论，当时用户希望能够在
+[minikube](https://github.com/istio/istio/issues/46163) 和
+[Docker Desktop](https://github.com/istio/istio/issues/47436) 上使用 Ambient 模式，
+希望使用 [Cilium](https://github.com/istio/istio/issues/44198)
+和 [Calico](https://github.com/istio/istio/issues/40973) 等 CNI 实现，
+还希望能够支持在 [OpenShift](https://github.com/istio/istio/issues/42341)
+和 [Amazon EKS](https://github.com/istio/istio/issues/42340) 等专用 CNI 实现上运行的服务。
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:22]:
```suggestion
和 [Amazon EKS](https://github.com/istio/istio/issues/42340) 等使用内部 CNI 实现上运行的服务。
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 11]
```diff
@@ -0,0 +1,264 @@
...
+这个主要 CNI 实现可能由您的云供应商（AKS、GKE 和 EKS 自己）提供，
+也可能由 Calico 和 Cilium 等第三方 CNI 实现提供。
+某些服务网格还可能捆绑了他们自己的主要 CNI 实现，他们明确要求只有这些实现才能让网格正常运行。
+
+基本上，在使用 mTLS 保护 Pod 流量并在服务网格层应用高级身份验证和授权策略等操作之前，
+您必须拥有一个具有功能性 CNI 实现的功能性 Kubernetes 集群，
+以确保设置基本网络路径以便数据包可以从集群中的一个 Pod 发送到另一个 Pod（以及从一个节点发送到另一个节点）。
+
+尽管某些服务网格也可能提供并需要自己的内部主要 CNI 实现，
+但有时可以在同一集群内并行运行两个主要 CNI 实现（例如，一个由云提供商提供，另一个由第三个实现），
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:50]:
```suggestion
而且有时可以在同一集群内并行运行两个主要 CNI 实现（例如，一个由云提供商提供，另一个由第三方实现），
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 12]
```diff
@@ -0,0 +1,264 @@
...
+实际上，这会引入一系列兼容性问题、奇怪的行为、功能集减少以及由于每个
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:51]:
```suggestion
但实际上，这会引入一系列兼容性问题、奇怪的行为、功能集减少以及由于每个
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 13]
```diff
@@ -0,0 +1,264 @@
...
+CNI 内部实现可能采用的很大机制差异而导致的一些不兼容性。
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:52]:
```suggestion
CNI 实现可能内部使用的机制差异而导致的一些不兼容性。
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 14]
```diff
@@ -0,0 +1,264 @@
...
+
+为了避免这种情况，Istio 项目选择不发布或要求我们提供自己的主要 CNI 实现，
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:54]:
```suggestion
为了避免这些问题，Istio 项目选择不发布或要求我们自己的主要 CNI 实现，
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 15]
```diff
@@ -0,0 +1,264 @@
...
+甚至不需要“首选” CNI 实现 - 而是选择使用尽可能广泛的 CNI 实现生态系统来支持 CNI 链，
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:55]:
```suggestion
甚至不需要“首选” CNI 实现 - 而是选择支持 CNI 链接与使用尽可能广泛的 CNI 实现生态系统，
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 16]
```diff
@@ -0,0 +1,264 @@
...
+通常用于移除[对 `NET_ADMIN` 和 `NET_RAW` 功能的兼容性要求](/zh/docs/ops/deployment/requirements/)以供用户将 Pod 部署到网格中。
+`istio-cni` 是 Ambient 数据平面模式中必需的组件。
+`istio-cni` 组件**不是**主要 CNI 实现，它是一个节点代理，可以对集群中已存在的任何主要 CNI 实现进行扩展。
+
+每当 Pod 被添加到 Ambient 网格时，`istio-cni` 组件都会为 Pod
+和 Pod 所在节点中运行的 [ztunnel](/zh/blog/2023/rust-based-ztunnel/)
+之间的所有传入和传出流量通过节点级网络命名空间配置流量重定向。
+Sidecar 机制和 Ambient Alpha 版机制之间的主要区别在于，在后者中，
+当 Pod 流量被重定向出 Pod 网络命名空间，并进入同位 ztunnel Pod
+网络命名空间时 - 途中必然经过主机网络命名空间，这是为实现批量流量重定向规则这一目标而实施的。
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:71]:
```suggestion
网络命名空间时 - 途中必然经过主机网络命名空间，这是实现此目标的批量流量重定向规则所在的地方。
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 17]
```diff
@@ -0,0 +1,264 @@
...
+使用这种方法在这些不同的环境中以通用的方式实现我们的目标根本不可行。
+
+在主机网络命名空间中重定向流量的根本问题在于，
+这正是集群的主要 CNI 实现**必须**配置流量路由/网络规则的地方。
+这造成了不可避免的冲突，最关键的是：
+
+- 主要 CNI 实现的基本主机级网络配置可能会干扰 Istio 的 CNI
+  扩展的主机级 Ambient 网络配置，从而导致流量中断和其他冲突。
+- 如果用户部署了由主要 CNI 实现强制执行的网络策略，
+  则在部署 Istio CNI 扩展时可能不会强制执行网络策略（取决于主要 CNI 实现如何强制执行 NetworkPolicy）
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:84]:
```suggestion
  则在部署 Istio CNI 扩展时可能不会强制执行网络策略（取决于主要 CNI 实现如何执行 NetworkPolicy）
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 18]
```diff
@@ -0,0 +1,264 @@
...
+
+在进行原型设计并充分验证这种新颖方法确实适用于我们可以访问的所有 Kubernetes 平台之后，
+我们对这项工作建立了信心，并决定将这一新的工作负载 Pod 和 ztunnel 节点代理组件之间的 **in-Pod**
+流量重定向机制模式贡献到上游，该机制是从头开始建立的，
+与所有主要云提供商和 CNI 高度兼容。
+
+关键的创新是将 Pod 的网络命名空间传递到同位的 ztunnel，
+以便 ztunnel 可以在 Pod 的网络命名空间内部启动其重定向套接字，同时仍然在 Pod 外部运行。
+通过这种方法，ztunnel 和应用程序 Pod 之间的流量重定向的方式与当今的 Sidecar 和应用程序 Pod 非常相似，
+并且对于在节点网络命名空间中运行的任何 Kubernetes 主要 CNI 都是严格不可见的。
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:115]:
```suggestion
并且对于在节点网络命名空间中运行的任何 Kubernetes 主要 CNI 完全不可见。
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 19]
```diff
@@ -0,0 +1,264 @@
...
+网络策略可以继续由任何 Kubernetes 主要 CNI 执行和管理，
+无论 CNI 使用 eBPF 还是 iptables，都不会发生任何冲突。
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:117]:
```suggestion
无论 CNI 是否使用 eBPF 或 iptables，都不会发生任何冲突。
```

**Review by hanxiaop — COMMENTED** [review]
too many words... I'm trying to read it carefully.

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 20]
```diff
@@ -0,0 +1,264 @@
...
+### Linux、Kubernetes 和 CNI - 什么是网络命名空间，为什么它很重要？ {#linux-kubernetes-and-cni----whats-a-network-namespace-and-why-does-it-matter}
+
+在 Linux 中，**容器**是在隔离的 Linux 命名空间中运行的一个或多个 Linux 进程。
+Linux 命名空间只是一个内核标志，用于控制在该命名空间内运行的进程能够看到的内容。
+例如，如果您通过 `ip netns add my-linux-netns` 命令创建一个新的 Linux 网络命名空间并在其中运行一个进程，
+则该进程只能看到在该网络命名空间中创建的网络规则。
+它看不到在其外部创建的任何网络规则 - 即使该计算机上运行的所有内容仍然共享一个 Linux 网络堆栈。
+
+Linux 命名空间在概念上很像 Kubernetes 命名空间 - 组织和隔离不同活动进程的逻辑标签，
+并允许您创建关于给定命名空间内的事物可以看到的规则以及对它们应用什么规则 - 它们只能以非常低的等级运行。
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:132]:
```suggestion
并允许您创建关于给定命名空间内的事物可以看到的规则以及对它们应用什么规则 - 它们只是在更低的层次上运行。
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 21]
```diff
@@ -0,0 +1,264 @@
...
+然后，CRI 调用 [容器**网络**接口（CNI）](https://github.com/containernetworking/cni)，
+该接口负责连接各个 Linux 网络命名空间中的网络规则，
+以便数据包离开和进入新的 Pod 就可以到达他们应该去的地方。
+对于 Kubernetes 或容器运行时来说，CNI 使用什么拓扑或机制来实现这一点并不重要 - 只要数据包到达它们应该在的地方，
+Kubernetes 正常工作，所有人都高兴。
+
+### 为什么我们放弃之前的模式？ {#why-did-we-drop-the-previous-model}
+
+在 Istio Ambient 网格中，每个节点至少有两个作为 Kubernetes DaemonSet 运行的容器：
+- 高效的 ztunnel，可作为网格流量代理职责和 L4 策略执行。
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:156]:
```suggestion
- 一个高效的 ztunnel，可作为网格流量代理职责和 L4 策略执行。
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 22]
```diff
@@ -0,0 +1,264 @@
...
+- 一个 `istio-cni` 节点代理，负责将新的和现有的 Pod 添加到 Ambient 网格中。
+
+在之前的 Ambient 网格实现中，应用程序 Pod 被添加到 Ambient 网格的方式如下：
+- `istio-cni` 节点代理检测现有或新启动的 Kubernetes Pod，
+  其命名空间被标记为 `istio.io/dataplane-mode=ambient`，表明它应包含在 Ambient 网格中。
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:161]:
```suggestion
  其命名空间被标记为 `istio.io/dataplane-mode=ambient`，表明它应被包含在 Ambient 网格中。
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 23]
```diff
@@ -0,0 +1,264 @@
...
+
+这意味着对于 Ambient 网格中 Pod 创建的数据包，该数据包将离开该源 Pod，
+进入节点的主机网络命名空间，然后理想情况下会被拦截并重定向到该节点的 ztunnel（在其自己的网络命名空间中运行）
+用于代理到目标 Pod，返回流程类似。
+
+这个模式作为初始 Ambient 网格 Alpha 版实现的简易设计工作得很好，
+但正如前面提到的，它有一个基本问题 - 具有许多 CNI 实现，并且在 Linux 中，
+有许多根本不同且不兼容的方法，您可以使用在其中配置数据包的方式从一个网络名称空间到另一个网络名称空间。
+您可以使用隧道、全覆盖网络、通过主机网络命名空间或绕过它。
+您可以浏览 Linux 用户空间网络堆栈，也可以跳过它并在内核空间堆栈中来回传输数据包，等等。
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:174]:
```suggestion
您可以通过 Linux 用户空间网络堆栈进行数据包处理，也可以跳过它并在内核空间堆栈中来回传输数据包，等等。
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 24]
```diff
@@ -0,0 +1,264 @@
...
+对于每种可能的方法，也许都有一个利用它的 CNI 实现。
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:175]:
```suggestion
对于每种可能的方法，也许都有一个 CNI 实现在使用它。
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 25]
```diff
@@ -0,0 +1,264 @@
...
+
+### Istio Ambient 流量重定向：新模式 {#istio-ambient-traffic-redirection-the-new-model}
+
+在新的 Ambient 模式中，应用程序 Pod 被添加到 Ambient 网格的方式如下：
+- `istio-cni` 节点代理检测到一个 Kubernetes Pod（现有的或新启动的），
+  其命名空间被标记为 `istio.io/dataplane-mode=ambient`，表明它应该包含在 Ambient 网格中。
+  - 如果启动了一个应添加到 Ambient 网格中的**新** Pod，
+    则 CRI 会触发 CNI 插件（由 `istio-cni` 代理安装和管理）。
+    该插件用于将新的 Pod 事件推送到节点的 `istio-cni` 代理，
+    并阻止 Pod 启动，直到代理成功配置重定向。由于 CNI 插件在 Kubernetes Pod 创建过程中尽可能早的被 CRI 调用，
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:192]:
```suggestion
    并阻止 Pod 启动，直到代理成功配置重定向。由于 CNI 插件由 CRI 尽早在 Kubernetes Pod 创建过程中调用，
```

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 26]
```diff
@@ -0,0 +1,264 @@
...
+    这确保了我们可以尽早建立流量重定向，以防止启动期间流量逃逸，而无需依赖初始化容器之类的机制。
```
Comment (hanxiaop) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:193]:
```suggestion
    这确保了我们可以足够早地建立流量重定向，以防止启动期间流量逃逸，而无需依赖初始化容器之类的机制。
```

**Review by hanxiaop — COMMENTED** [review]
Everything else LGTM, incredible speed. 🚀

**Comment by windsonsea** [comment]
Shall we request the blog author to take a look?
cc @linsun

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md [hunk 27]
```diff
@@ -0,0 +1,264 @@
...
+以帮助 Ambient 更快进阶至 Beta，让更多用户可以在 Istio 1.21 或更高版本中用上 Ambient，
+并在他们各自的平台中体验 Ambient 这种无 Sidecar 网格的优势，不用再操心现有的 CNI 或首选的 CNI 是什么。
+
+## 回首来时路，披荆斩棘 {#how-did-we-get-here}
+
+### 服务网格和 CNI：错综复杂 {#service-meshes-and-cnis-its-complicated}
+
+Istio 是一种服务网格，而所有服务网格严格意义上来说都不是
+**CNI 实现** ，服务网格想要在所有 Kubernetes 集群运行，
+底层需要一个[合规的主要 CNI 实现](https://www.cni.dev/docs/spec/#overview-1)。
```
Comment (nicole-lihui) [inline content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:39]: `primary CNI` 是否考虑用 `主流 CNI` 来替换 `主要 CNI` 更加合适呢?

Comment (windsonsea) [reply content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md:39]: yes, it's better to use 主流 CNI

**Comment by linsun** [comment]
> Shall we request the blog author to take a look? cc @linsun

While I'm fluent in Chinese, my Chinese is not good on IT words. I trust your judgement :)

File: content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md (hunk 27 above)
Comment (wilsonwu) [reply content/zh/blog/2024/inpod-traffic-redirection-ambient/index.md]: Done

**Comment by windsonsea** [comment]
OK, thanks for your great work.

**Review by windsonsea — APPROVED** [review]
Thanks, lgtm

**Pull request merged**